### PR TITLE
SA-4 | All changes till now including correct migration files

### PR DESCRIPTION
**Pull request by vanshksharma**

### CODE REVIEW

**Comment by vanshksharma** [comment]
@samay-rgb @PranshuNayak Due to some unknown errors Migration files were not working correctly and I merged them in one file and now it is working. Kindly approve

**Comment by vanshksharma** [comment]
@akshatnema Kindly merge the PR

**Comment by akshatnema** [comment]
@PranshuNayak @samay-rgb Kindly review the PR.

**Review by rishiktejatanugula — COMMENTED** [review]
Approve @akshatnema

**Comment by PranshuNayak** [comment]
@vanshksharma push the changes of models.py too for which these migrations are generated, or reference the PR where you have done those.

**Review by PranshuNayak — APPROVED** [review]
@akshatnema migrations file i.e `FusionIIIT/applications/placement_cell/migrations/0002_auto_20230407_1155.py`
only need to be pushed.

**Review by samay-rgb — APPROVED** [review]

**Comment by akshatnema** [comment]
Changes in the migrations file. Can you approve it again @PranshuNayak ?

**Comment by vanshksharma** [comment]
@akshatnema These Changes were made because @PranshuNayak told me to do so. Only One line has been changed and that too was a comment. Somehow it got changed and I reverted it back. You can Safely merge the PR, but be sure to merge it because we have to show it tomorrow.

File: FusionIIIT/applications/placement_cell/migrations/0002_auto_20230407_1155.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        ('academic_information', '0002_student_hall_id'),
+        ('placement_cell', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='placementrecord',
+            name='branch',
+            field=models.CharField(default='', max_length=100),
+        ),
```
Comment (PranshuNayak) [inline FusionIIIT/applications/placement_cell/migrations/0002_auto_20230407_1155.py]: @vanshksharma the migrations for branch are already present on the test you can see them [here](https://github.com/FusionIIIT/Fusion/blob/test/FusionIIIT/applications/placement_cell/migrations/0003_placementrecord_branch.py). Remove these lines here.

File: FusionIIIT/applications/placement_cell/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.1.5 on 2023-04-06 12:31
+# Generated by Django 3.1.5 on 2023-03-15 18:53
```
Comment (PranshuNayak) [inline FusionIIIT/applications/placement_cell/migrations/0001_initial.py]: You don't need to push this file , while you push you add files in your PR using git add ignore this file.

**Comment by vanshksharma** [comment]
@PranshuNayak kindly check the changes

**Review by PranshuNayak — APPROVED** [review]
@akshatnema changes are fine ,only push the migration file `FusionIIIT/applications/placement_cell/migrations/0002_auto_20230407_1155.py` from the latest commit

**Comment by PranshuNayak** [comment]
@samay-rgb please go through the changes done in the latest commit .

**Comment by whhygee** [comment]
@vanshksharma To handle the large number of commits in the thread, please squash them going ahead

**Pull request closed without merging**